stores.html

```diff
@@ -5,8 +5,164 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./css/stores.css" />
-    <title>Document</title>
+    <title>Store | Karma</title>
   </head>
 
-  <body></body>
+  <body>
+    <header class="header">
+      <nav class="header_nav">
+        <div class="logo_div">
+          <img class="logo" src="img/karma-logo.svg" alt="logo" />
+        </div>
+        <div class="list_div">
+          <ul class="header_list">
+            <li class="list_item">
+              <a class="nav_link" href="./index.html">Meet Karma</a>
+            </li>
+            <li class="list_item">
+              <a class="nav_link" href="#how_it_works">How it Works</a>
+            </li>
+            <li class="list_item">
+              <a class="nav_link" href="./stores.html">Stores</a>
+            </li>
+            <li class="list_item">
+              <a class="nav_link" href="#blog">Blog</a>
+            </li>
+            <li class="list_item">
+              <a class="nav_link" href="#help">Help</a>
+            </li>
+            <li class="list_item">
+              <a class="nav_link" href="#log_in">Log in</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+
+    <main class="main">
+      <article class="store">
+        <section class="store_form">
+          <h1 style="color: orangered;">
+            Order your Karma wifi device today!
+          </h1>
+
+          <div class="name_item">
+            <label for="fist_name"
+              >First name*
+              <br />
+              <input type="text" class="input" required />
```

Comment:
Watch out, `<input>` elements with missing `name` attribute and won't be submitted